integration_tests/dbt_project.yml

```diff
@@ -61,7 +61,7 @@ vars:
   # ensure integration tests run successfully when there are 0 of a given model type (extra)
   exclude_packages: ['exclude_package']
   exclude_paths_from_project: ["/to_exclude/","source_3.table_6"]
-  model_types: ['staging', 'intermediate', 'marts', 'other', 'extra', 'new_model_type']
+  model_types: ['base','staging', 'intermediate', 'marts', 'other', 'extra', 'new_model_type']
```

Comment:
This is just for the integration test right?

Reply:
yep -- that's what happens when you commit from inside a subdirectory 🤦